gen.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Running `make generate` should do the same as this script, is there a good reason to add it?

Reply:
I didn't know that there is `generate` in the Makefile. So the script is useless
I have made a new commit for that